eng/targetingpacks.targets

```diff
@@ -87,6 +87,16 @@
     </PropertyGroup>
   </Target>
 
+  <!-- SDK tries to download runtime packs when RuntimeIdentifier is set, remove them from PackageDownload item. -->
+  <Target Name="RemoveRuntimePackFromDownloadItem"
+          Condition="'$(_UseLocalTargetingRuntimePack)' == 'true' and '$(RuntimeIdentifier)' != ''"
```

Comment:
I would remove the RuntimeIdentifier condition just to be on the safe side to prevent accidental runtime downloads in the future in case the sdk doesn't just condition on RuntimeIdentifier.

Reply:
Sounds good.